Auto3dgm.json

```
@@ -1,8 +1,9 @@
 {
-  "$schema": "https://raw.githubusercontent.com/Slicer/Slicer/main/Schemas/slicer-extension-catalog-entry-schema-v1.0.0.json#",
+  "$schema": "https://raw.githubusercontent.com/Slicer/Slicer/main/Schemas/slicer-extension-catalog-entry-schema-v1.0.1.json#",
   "build_dependencies": [],
   "build_subdirectory": ".",
   "category": "Auto3dgm",
   "scm_revision": "master",
-  "scm_url": "https://github.com/ToothAndClaw/SlicerAuto3dgm"
+  "scm_url": "https://github.com/ToothAndClaw/SlicerAuto3dgm",
+  "tier": 3
```

Based on https://github.com/Slicer/ExtensionsIndex/issues/2050, the developers are no longer supporting it, often it fails and an indication to remove it from the Slicer extensions index entirely:
```suggestion
  "tier": 1
```